dvc/repo/experiments/queue/tempdir.py

```diff
@@ -77,9 +77,8 @@ def iter_active(self) -> Generator[QueueEntry, None, None]:
         for stash_rev in self.proc:
             infofile = self.get_infofile_path(stash_rev)
             executor_info = ExecutorInfo.load_json(infofile)
-            if (
```

Comment:
This is a bug, which prevents we get a live matrix during `temp_dir` experiment running.